src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -120,25 +134,26 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
             return;
         }
 
-        if (files[0].endsWith(".json")) {
-            return await this.importAlternativeManagerProfile(files[0]);
-        }
-
         let read: string | null = await this.readProfileFile(files[0]);
 
         if (read !== null) {
-            const parsed: ExportFormat = await this.parseYamlToExportFormat(read);
-
-            const localListenerId = this.listenerId + 1;
-            this.listenerId = localListenerId;
-            document.addEventListener('created-profile', ((event: CustomEvent) =>
```

Comment:
Moving this stuff to `installProfileHandler` makes the comment above this function outdated.